### ISSUE TITLE
camelcase rule considers interface properties as names

### ISSUE THREAD

**Issue by mightyiam** [mightyiam/eslint-config-standard-with-typescript#3]
So complains about them even when rule is set with `properties: 'never'`.

So, try
```
interface Foo {
  b_ar: string // eslint error: camelcase
}

```

**Comment by guylil** [comment]
https://github.com/nzakas/eslint-plugin-typescript/pull/95

#goodnessSquad

**Comment by mightyiam** [comment]
Fixed in #54.
